cms/bundles/utils.py

```diff
@@ -170,6 +186,17 @@ def get_preview_items_for_bundle(bundle: "Bundle", page_id: int, pages_in_bundle
     return preview_items
 
 
+def get_release_calendar_page_title_with_status_and_release_date(
+    release_calendar_page: "Page",
+) -> str:
+    """Returns the release page title, status and release date."""
+    return (
+        f"{release_calendar_page.specific_deferred.title}",
+        f" ({release_calendar_page.specific_deferred.get_status_display()},"
```

Comment:
We might have to handle better here for statuses, as I mentioned previously. You can dig into the release calendar statuses to see this. Will take a look as well.